### DISCUSSION TITLE
文档里面提到了'Canvas 渲染', 是不是figma就是Canvas 渲染, 那么怎么学习Canvas 渲染,哪边有资料呀.

### DISCUSSION THREAD

**Discussion by LukerSpringtree** [pipipi-pikachu/PPTist#253] [General]
文档里面提到了'Canvas 渲染', 是不是figma就是Canvas 渲染, 那么怎么学习Canvas 渲染,哪边有资料呀.

**Comment by pipipi-pikachu** [comment]
是，想了解canvas做图形编辑，直接在各种开发者社区搜“低代码”、“在线设计”、“海报设计”之类的就行了，基本全部都是用canvas渲染的，不过基本全部是基于Fabric.js或者Konva.js这些canvas库来做的

**Reply by LukerSpringtree** [reply]
学习一下
